hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -487,15 +487,6 @@
   final static String UNSAFE_RS_HOSTNAME_DISABLE_MASTER_REVERSEDNS_KEY =
     "hbase.unsafe.regionserver.hostname.disable.master.reversedns";
 
-  /**
-   * HBASE-24667: This config hbase.regionserver.hostname.disable.master.reversedns will be replaced by
```

Comment:
The original author made an effort to document that this deprecation happened as part of HBASE-24667, please preserve this in a comment adjacent to the config's `DeprecationDelta`.